### ISSUE TITLE
Super simple fix for LineChart data. Please fix asap. itll take seconds!

### ISSUE THREAD

**Issue by descyther** [Abhinandan-Kushwaha/react-native-gifted-charts#490]
The data3 prop is broken on LineCharts. It overlaps data2. The fix is in LineCharts index.ts and its super easy, please fix it or ill make a PR if i get time.

Literally this code is where the break is.

```
        if (data2.length && i >= startIndex2 && i <= endIndex2) {
            if (stepChart || stepChart2) {
              pp2 += getStepPath(data2, i);
              (' ');
            } else {
              pp2 += getSegmentPath(
                data2,
                i,
                lineSegments2,
                startIndex2,
                endIndex2,
              );
            }
          }
          if (data3.length && i >= startIndex3 && i <= endIndex3) {
            if (stepChart || stepChart3) {
              pp3 += getStepPath(data3, i);
            } else {
              pp3 += getSegmentPath(
                data2,                                             <----- needs changing to data3 please!!!
                i,
                lineSegments2,
                startIndex3,
                endIndex3,
              );
            }
          }
```

**Comment by Abhinandan-Kushwaha** [comment]
Merged your PR and released version `1.3.30`

**Comment by descyther** [comment]
legend, works like a charm too. Glad it was resolved!
